airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeSqlOperations.java

```diff
@@ -46,7 +46,7 @@ public void createTableIfNotExists(JdbcDatabase database, String schemaName, Str
     final String createTableQuery = String.format(
         "CREATE TABLE IF NOT EXISTS %s.%s ( \n"
             + "%s VARCHAR PRIMARY KEY,\n"
-            + "\"%s\" VARIANT,\n"
+            + "%s VARIANT,\n"
```

Comment:
why is this changing?

Reply:
All destinations are producing without quotes and case insensitive columns but I think I need to review a bit the logic around column identifiers and quoting anyway...